downloader/src/main/kotlin/Downloader.kt

```diff
@@ -77,7 +80,39 @@ object Downloader {
 
         val exception = DownloadException("Download failed for '${pkg.id.toCoordinates()}'.")
 
-        // Try downloading from VCS.
+        var provenance: Provenance? = null
```

Comment:
The construct with the mutable *var* can probably be avoided by using `fold()` instead of `forEach()`.

Reply:
Resolved with @sschuberth's [suggestion](https://github.com/oss-review-toolkit/ort/pull/3736#discussion_r596886120)